js/modules/jake.js

```diff
@@ -0,0 +1,111 @@
+export default {
+    state: {
+        profile: {
+            username: "jake18",
```

Comment:
Put your correct GitHub username here.

```suggestion
            username: "jake018",
```